core/api/graphql/query_artifact_files_by_manifest_entries.graphql

```diff
@@ -0,0 +1,27 @@
+query ArtifactFileURLsByManifestEntries(
+  $id: ID!
+  $entries: [ArtifactManifestEntryInput!]
+  $storageLayout: String!
+  $manifestVersion: String!
+  $storageRegion: String
+) {
+  artifact(id: $id) {
+    filesByManifestEntries(
```

Comment:
Has the PR to add this query been opened yet? Would love to take a look!

Reply:
yes it's open, will send it to you!